examples/cloudflare-pages/app/routes/_index.tsx

```diff
@@ -4,11 +4,12 @@ import { useLoaderData } from '@remix-run/react'
 export const loader = (args: LoaderFunctionArgs) => {
   const extra = args.context.extra
   const cloudflare = args.context.cloudflare
-  return { cloudflare, extra }
+  const myVarInVariables = args.context.hono.context.get('MY_VAR_IN_VARIABLES')
```

Comment:
This `args.context.hono.context` seems verbose, but I don't have a good idea for the naming.